pkg/registry/apis/query/query.go

```diff
@@ -371,6 +375,25 @@ func (b *QueryAPIBuilder) handleExpressions(ctx context.Context, req parsedReque
 	return qdr, nil
 }
 
+func (b *QueryAPIBuilder) convertQueryWithoutExpression(ctx context.Context, req datasourceRequest,
+	qdr *backend.QueryDataResponse) (*backend.QueryDataResponse, error) {
+	allowLongFrames := false
+	refID := req.Request.Queries[0].RefID
```

Comment:
should the length of `Queries` be checked here, or is it safe?

Reply:
Yeah, better safe than sorry. Added a couple of checks here.